sqlserver/metadata.csv

```diff
@@ -60,11 +60,11 @@ sqlserver.database.avg_fragmentation_in_percent,gauge,,,,"Logical fragmentation
 sqlserver.database.backup_count,gauge,,,,"The total count of successful backups made for a database.",0,sql_server,backup count,
 sqlserver.database.fragment_count,gauge,,,,"The number of fragments in the leaf level of an IN_ROW_DATA allocation unit.",0,sql_server,fragment count,
 sqlserver.database.is_sync_with_backup,gauge,,,,"Whether or not the database is marked for replication synchronization with backup. 0 = Not marked for replication sync, 1 = Marked for replication sync.",0,sql_server,is sync with backup,
-sqlserver.database.file_space_usage.free_space,gauge,,mebibyte,,The amount of free space in the database file,0,sql_server,database file free space,
-sqlserver.database.file_space_usage.used_space_by_version_store,gauge,,mebibyte,,The amount of space used by the version store in the database file,0,sql_server,database file used space by version store,
-sqlserver.database.file_space_usage.used_space_by_internal_object,gauge,,mebibyte,,The amount of space used by internal objects in the database file,0,sql_server,database file used space by internal object,
-sqlserver.database.file_space_usage.used_space_by_user_object,gauge,,mebibyte,,The amount of space used by user objects in the database file,0,sql_server,database file used space by user object,
-sqlserver.database.file_space_usage.mixed_extent_space,gauge,,mebibyte,,The amount of space used by mixed extents in the database file,0,sql_server,database file mixed extent space,
+sqlserver.tempdb.file_space_usage.free_space,gauge,,mebibyte,,The amount of free space in the tempdb database file,0,sql_server,tempdb database file free space,
+sqlserver.tempdb.file_space_usage.used_space_by_version_store,gauge,,mebibyte,,The amount of space used by the version store in the tempdb database file,0,sql_server,tempdb database file used space by version store,
```

Comment:
nit:

Because we have the prefix `file_space_usage`, the suffix can just be `version_store_space`, `internal_object_space`, `mixed_extent_space`, `user_object_space`.